use_cases/realtime_analytics.rst

```diff
@@ -160,31 +160,39 @@ The following function wraps the rollup query up for convenience.
 
 .. code-block:: plpgsql
 
+  -- single-row table to store when we rolled up last
```

Comment:
Really [schema variables](http://www.postgresql-archive.org/proposal-schema-variables-td5989266.html) would be better, but they're not yet available.

Reply:
Actually there's another technique: directly inspect the timestamp of the last insertion commit.

```sql
select pg_xact_commit_timestamp(xmin) from http_request_1min;
```

Then we wouldn't have to maintain our own metadata table...